__tests__/index.test.js

```diff
@@ -130,6 +130,17 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 
+  test("GET 200: Results are sorted by comments in a descending order", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.rows).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
```

Comment:
We need a test on this endpoint to check what happens when
- 404 non-existant article_id, eg. `/api/articles/9999/comments`
- 40 invalid article_id, eg. `/api/articles/katherine/comments`